extension/src/content/content.js

```diff
@@ -7,105 +7,136 @@ window.addEventListener("load", () => {
   const form = document.querySelector("form");
 
   // create new elements
-  const ghostTextarea = document.createElement("div");
-  ghostTextarea.classList.add("ghost-textarea");
-  ghostTextarea.setAttribute("contenteditable", "true");
-  parent.appendChild(ghostTextarea);
-  // textarea and ghostTextarea are now siblings
+  const wiseTextarea = document.createElement("div");
+  wiseTextarea.classList.add("wise-textarea");
+  wiseTextarea.setAttribute("contenteditable", "true");
+  parent.appendChild(wiseTextarea);
```

Comment:
Changed naming system to wiseTextarea (which contains ghostText and autocompleteText) and parent.